src/main/java/de/zalando/aruha/nakadi/validation/ValidationStrategy.java

```diff
@@ -0,0 +1,27 @@
+package de.zalando.aruha.nakadi.validation;
+
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Maps;
+
+import de.zalando.aruha.nakadi.domain.EventType;
+import de.zalando.aruha.nakadi.domain.ValidationStrategyConfiguration;
+
+public abstract class ValidationStrategy {
+
+    private static final Map<String, ValidationStrategy> STRATEGIES = Maps.newHashMap();
+
+    static {
+        STRATEGIES.put(EventBodyMustRespectSchema.NAME, new EventBodyMustRespectSchema());
+        STRATEGIES.put(FieldNameMustBeSet.NAME, new FieldNameMustBeSet());
+    }
+
+    public abstract EventValidator materialize(EventType eventType, ValidationStrategyConfiguration vsc);
```

Comment:
With it a ValidationStrategy (something abstract) is materialized (made concrete) into a Validator.